src/components/List.css

```diff
@@ -1,3 +1,11 @@
+.list-container {
```

Comment:
I had to widen the top level div for the List and Add Item page so the header would span the entire width of the page

Reply:
Another way to possibly tackle that would be to create a Layout that wraps around App and put the header and footer in the layout. But I think if the header issue in the mobile view is resolved based on what you and Lauren talked about below then that's unnecessary.